src/components/TodoList/TodoList.tsx

```diff
@@ -1,12 +1,25 @@
 /* eslint-disable */
 import React from 'react';
+import { selector } from '../../app/store';
+import { useDispatch } from 'react-redux';
+import { filtredTodos } from '../../features/filter';
+import { setCurrentTodo } from '../../features/currentTodo';
 
 export const TodoList: React.FC = () => {
+  const filter = selector(state => state.filter);
+  const todos = selector(state => state.todos);
+  const currentTodo = selector(state => state.currentTodo);
+  const dispatch = useDispatch();
+
+  const filtredArrayOfTodos = filtredTodos(todos, filter);
```

Comment:
Ensure that the `filtredTodos` function is correctly implemented and imported. This function should filter the todos based on the current filter criteria.